attrd/commands.c

```diff
@@ -669,14 +669,41 @@ attrd_peer_update(crm_node_t *peer, xmlNode *xml, const char *host, bool filter)
 {
     bool changed = FALSE;
     attribute_value_t *v = NULL;
+    int dampen = 0;
 
+    const char *op = crm_element_value(xml, F_ATTRD_TASK);
     const char *attr = crm_element_value(xml, F_ATTRD_ATTRIBUTE);
     const char *value = crm_element_value(xml, F_ATTRD_VALUE);
+    const char *dvalue = crm_element_value(xml, F_ATTRD_DAMPEN);
 
     attribute_t *a = g_hash_table_lookup(attributes, attr);
 
     if(a == NULL) {
-        a = create_attribute(xml);
+        if (safe_str_eq(op, ATTRD_OP_UPDATE) || safe_str_eq(op, ATTRD_OP_UPDATE_VALUE_AND_DAMPEN)) {
+            a = create_attribute(xml);
+        }
```

Comment:
Hi Ken,

Okay.
I add the following correction.
`
        if (safe_str_eq(op, ATTRD_OP_UPDATE_ONLY_DAMPEN)) {
            crm_notice("Update error (attribute %s not found)", attr);
            return;
        }
`
Is my understanding wrong?